torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1885,6 +1888,24 @@ def forced_specializations(self):
             if s in self._marked_dynamic
         ])
 
+    def remove_redundant_dynamic_results(self):
```

Comment:
a bit of comment explaining its behavior would be nice.

Reply:
yeah, added